### ISSUE TITLE
1fichier com configuration with premium account

### ISSUE THREAD

**Issue by mthbyd** [pyload/pyload#1916]
Hi,

I have just discovered pyload! It seems to be a great tool. I do not know much about python. 
Could you please help me to understand how enable module onefichier.com and configure my premium account?

I'm using pyLoad 0.4.9 on a Debian.

Thank you for your help

**Comment by vuolter** [comment]
Firstly, take a look on the wiki (http://pyload.org/wiki) or in the forum (http://forum.pyload.org).

**Comment by mthbyd** [comment]
Before asking I have already checked the wiki and the forum.

When I try to add an account I'm not able to seen onefichier.com. In the folder module/plugin/hoster I have found the onefichier.com.py file but I do not understand how load/enable it and how configure my premium account. When I try to add a package with an URL. process is starting and ending in the same second without any download.

If you have any idea or another documentation it would be great.

Thanks
